app/config/parameters.yml.dist

```diff
@@ -8,12 +8,21 @@ parameters:
     database_path:     "%kernel.root_dir%/app.sqlite"
 
     phpcr_backend:
-        # use Jackalope Doctrine DBAL
+        # if you're using Jackalope Doctrine DBAL
         type: doctrinedbal
         connection: default
         caches:
             meta: doctrine_cache.providers.phpcr_meta
             nodes: doctrine_cache.providers.phpcr_nodes
+        
+        # if you're using Jackalope Jackrabbit, comment the lines before
+        # and uncomment the lines after this comment
+        #logging: true
+        #profiling: true
+        #type: jackrabbit
+        #url: http://localhost:8080/server/
+        #parameters:
+        #    jackalope.check_login_on_server: false
```

Comment:
since we are now lazy loading the Doctrine DBAL connection .. do we even need it for DBAL starting with the next release? that being said .. for performance I would still argue that its better to have it off, which is afaik the default in non debug mode.